src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -415,25 +442,49 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
             text: {
                 // NOTE: Check for tagName being a string, because it may be a function in an Angular app (T175340).
-                condition: el => typeof el.tagName === 'string' && domUtils.isScriptElement(el),
-                get:       el => typeof el.text === 'string' ? removeProcessingHeader(el.text) : el.text,
+                condition: el => typeof el.tagName === 'string' && typeof el.text === 'string' &&
+                                 (domUtils.isScriptElement(el) || domUtils.isStyleElement(el)),
 
-                set: (el, script) => {
-                    el.text = script ? processScript(script, true, false) : script;
+                get: el => {
+                    if (domUtils.isScriptElement(el))
+                        return getStoredWriteContent(el) || removeProcessingHeader(el.text);
+                    else if (domUtils.isStyleElement(el))
+                        return getStoredWriteContent(el) || styleProcessor.cleanUp(el.text, urlUtils.parseProxyUrl);
+                },
+
+                set: (el, text) => {
+                    if (text) {
+                        if (domUtils.isScriptElement(el))
+                            el.text = processScript(text, true, false);
+                        else if (domUtils.isStyleElement(el))
+                            el.text = styleProcessor.process(text, urlUtils.getProxyUrl);
+                    }
 
-                    return script;
+                    return text;
                 }
             },
 
             textContent: {
                 // NOTE: Check for tagName being a string, because it may be a function in an Angular app (T175340).
-                condition: el => typeof el.tagName === 'string' && domUtils.isScriptElement(el),
-                get:       el => typeof el.textContent === 'string' ? removeProcessingHeader(el.textContent) : el.textContent,
+                condition: el => typeof el.tagName === 'string' && typeof el.textContent === 'string' &&
+                                 (domUtils.isScriptElement(el) || domUtils.isStyleElement(el)),
```

Comment:
`typeof el.tagName === 'string' && typeof el.textContent === 'string'`

is it necessary ?